Bottom: 01e550ecebaf1ee55350d9c6878fc3b5a9aaf54c
Top:    bd7c5cc24827b29369c18cf5e25b1a28c8ea791b
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-05-28 15:21:54 +0800

Refresh of new-road

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 79999a6..d21177b 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2425,8 +2425,7 @@ int vhost_prefetch_heads(struct vhost_virtqueue *vq,
 
 	for (i = 0; i < total; i++) {
 		if (unlikely(vhost_get_avail(vq, heads[i].id,
-							&vq->avail->ring[(last_avail_idx
-		+ i)& (vq->num - 1)]))) {
+		    &vq->avail->ring[(last_avail_idx + i)& (vq->num - 1)]))) {
 			vq_err(vq, "Failed to get descriptors\n");
 			return -EFAULT;
 		}
